situp-plugins/apmtracesource/src/main/java/com/amazon/situp/plugins/health/HealthGrpcService.java

```diff
@@ -0,0 +1,26 @@
+package com.amazon.situp.plugins.health;
+
+import io.grpc.health.v1.HealthCheckRequest;
+import io.grpc.health.v1.HealthCheckResponse;
+import io.grpc.health.v1.HealthGrpc;
+import io.grpc.stub.StreamObserver;
+
+public class HealthGrpcService extends HealthGrpc.HealthImplBase {
+
+    @Override
+    public void check(final HealthCheckRequest request, final StreamObserver<HealthCheckResponse> responseObserver) {
+        if (isHealthy()) {
+            responseObserver.onNext(
+                    HealthCheckResponse.newBuilder().setStatus(HealthCheckResponse.ServingStatus.SERVING).build());
+        } else {
+            responseObserver.onNext(
```

Comment:
QUES: how is this block reached if isHealthy is always true? Is there something in the TODO?

Reply:
Yeah. Just for visibility to the whole team. I am OK with either way.